.github/workflows/build.yml

```diff
@@ -8,27 +8,29 @@ env:
 
 jobs:
   build-web:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
         name: Clone
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v3
         name: Setup Node
         with:
-          node-version: "*"
-      - name: Install Yarn
-        run: corepack enable
+          node-version: 16
```

Comment:
Why 16 not `lts/*`? I'd be fine with using the latest LTS (although this is ok too)